txp/Pos/Txp/Toil/Utxo/Functions.hs

```diff
@@ -157,23 +153,7 @@ verifyOutputs VTxContext {..} (TxAux UnsafeTx {..} _ distrs) =
           , sformat ("output #"%int%" sends money to a redeem address ("
                     %addressF%"), this is prohibited") i addr
           )
-        ] ++ checkDist i d txOutValue
-    checkDist i d txOutValue =
```

Comment:
Where do these checks live now, if anywhere?

Reply:
In `mkMultiKeyDistr`.